### PR TITLE
community/aircrack-ng: update apkbuild, clarify license

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

File: community/aircrack-ng/APKBUILD [hunk 1]
```diff
@@ -22,25 +23,21 @@ prepare() {
...
-	make check
-}
-
 build() {
 	cd "$builddir"
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
-		sqlite=true experimental=true \
```
Comment (jirutka) [inline community/aircrack-ng/APKBUILD]: Why did you remove sqlite=true?

Comment (jirutka) [reply community/aircrack-ng/APKBUILD]: Okay, then document it in the commit message.

Comment (TBK) [reply community/aircrack-ng/APKBUILD]: @jirutka Done.
